### PR TITLE
PR: Library v1.1

### PR DESCRIPTION
**Pull request by pastorp3**
This project was created to store books that the user can input through a form. It has the following features:

- The form is hidden unless the user clicks the add book button.
- When a book is added to the list is shown in a card below the form and the add book button
- Each book has the option to delete it from the list or to mark it as read.
- When you delete a book it is removed from the list forever.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,47 @@
...
+The user can mark a book as read or they can delete it from the list completely. Each book is shown in its own card with the buttons for mark as read and for delete the book.
+
+## Built With
+
+- Javascript,
+- HTML-CSS
+
+## Live Demo
+
+[Live Demo Link](https://raw.githack.com/pastorp3/Library/feature/index.html)
```
Comment (lekegitrepo) [inline README.md]: - **_Improve the readme instructions: include how to setup the project locally_**

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
## Status: Request changes :recycle:
### Well done :muscle: you have done a good job so far, however, you need to implement the following :point_down: requirements before this project can be approved.

- **_The form should allow to mark the Book status as read_**

- **_Each `Book` displayed should have a `button` to `toggle` the `Book` read status ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/toggle_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/library))_**

### Optional:
- **_The form used to create a book should have validations, you should not be able to create an empty book_**

#### _Reviewed by Temitayo Adeleke_

**Comment by pastorp3** [comment]
Add review changes:

-toggle-button for read option
-setup section to readme file

**Review by davidmrtz-dev — APPROVED** [review]
****
## Hi Team. 😄 
****
Great Job 💯 👍  You correctly implemented the required changes.
You can move onto the next project. 🎊 
Happy coding. 👨‍🚀  
****
### Status: Approved. ✅
****

Happy coding!  💻 
**[TSE:  David Elí]** 👨‍💻 
Slack: @ David Eli Martinez Garcia

**Pull request merged**